client/src/api/api_client.ts

```diff
@@ -51,8 +51,18 @@ class ApiClient {
   getGeoIds = async (lat: string, long: string): Promise<ApiResponse> => {
     return this.request(`${ApiClient.API_URL}/geolocate/${lat},${long}`, (data) => {
       return {
-        pwsId: data?.data?.water_system_pws_id,
-        zipCode: data?.data?.zip_code,
+        address: {
```

Comment:
right now if there is an address search we trigger getParcel if the geotype is address, but we could use this and only do the parcel search if there is an address present: https://github.com/BlueConduit/open-data-platform/blob/main/client/src/components/PredictionPanel.vue#L133 

would save an extra call to the API always since we can avoid doing water system fetch when an address is present on geoids too

Reply:
Done!